app.py

```diff
@@ -0,0 +1,142 @@
+            )
+
+        set_token(r.json().get('access_token'))
+
+        return render_template('main.html', account={})   
+    except:
+        return render_template('index.html', AUTH_URL = AUTH_URL, LOGOUT_URL=LOGOUT_URL)   
+
+
+@app.route('/account', methods=['POST','GET'])
+def account():
+
+    if request.method == 'POST':
+        requestURL = "https://" + os.getenv('REGION') + ".api.riotgames.com/riot/account/v1/accounts/me"
+        header = {"Authorization": 'Bearer ' + token}
+
+        try:
+            response = requests.get(requestURL, headers=header)
+
+            if response.status_code != 200: 
+                return render_template('main.html', ACCOUNT_INFO = response.json().get('status')["message"])
+
+            
+            return render_template('main.html', ACCOUNT_INFO = response.json())
+        except:
+
+            return render_template('main.html', ACCOUNT_INFO = "Error getting account info ")
+
+
+@app.route('/accounts/me', methods=['GET'])
+def accountMe():
+
+    ## validate token 
+    if token != "": 
+
+        requestURL = "https://" + os.getenv('REGION') + ".api.riotgames.com/riot/account/v1/accounts/me"
+        header = {"Authorization": 'Bearer ' + token}
+
+        try:
+            response = requests.get(requestURL, headers=header)
+
+            if response.status_code != 200: 
+                return jsonify("Not data was found"), 401  
+
+            return response.json(), 200  
+        except:
+
+            return jsonify("Internal Server Error"), 500  
+
+    return jsonify("Authentication process is missing"), 500   
+
+
+@app.route('/summoners/overview', methods=['GET'])
+def summonersOverview():
+
+    region  = request.args.get('region', None)
+    name  = request.args.get('name', None)
+
+    ## validate token 
+    if token != "": 
+
+        requestSummonerURL = "https://" + region + ".api.riotgames.com/lol/summoner/v4/summoners/by-name/" + name
+        requestSummonerInfoURL = "https://" + region + ".api.riotgames.com/lol/league/v4/entries/by-summoner/"
+        header = {"X-Riot-Token": os.getenv('APP_KEY')}
+
+        try:
+            response = requests.get(requestSummonerURL,headers=header)
+            
+            summoner = models.Summoner(
+                response.json().get("id"),
+                response.json().get("name"),
+                response.json().get("profileIconId"),
+                response.json().get("summonerLevel"),
+                datetime.utcfromtimestamp(response.json().get("revisionDate")/ 1000.0),
+                []
+            )
+
+            requestSummonerInfoURL += summoner.id
+            responseEntries = requests.get(requestSummonerInfoURL,headers=header)
+
+            summoner.leagueEntries = responseEntries.json()
+
+            return json.dumps(vars(summoner),indent=0, sort_keys=True, default=str), 200  
+        except Exception as e:
+            print(e)
```

Comment:
Let's be consistent on whether we print exceptions

Reply:
done